helper_methods.rb

```diff
@@ -292,3 +292,25 @@ def add_revision_statement(uri, description)
   post = @client.post(uri, resource.to_json)
   puts post.body
 end
+
+def get_all_resource_uris_for_institution()
+  #run through all repositories
+  resources_endpoints = []
+  repos_all = (3..12).to_a
+  repos_all.each do |repo|
```

Comment:
This might be an interesting case to lean about map.  This could be written as
```
resources_endpoints = repos_all.map do |repo|
                        "repositories/#{repo}/resources"
                      end
```

Reply:
Cool! Thanks!